ui/apps/platform/src/types/featureFlag.ts

```diff
@@ -6,4 +6,5 @@ export type FeatureFlagEnvVar =
     | 'ROX_COMPLIANCE_ENHANCEMENTS'
     | 'ROX_VULN_MGMT_REPORTING_ENHANCEMENTS'
     | 'ROX_VULN_MGMT_WORKLOAD_CVES'
+    | 'ROX_CENTRAL_EVENTS'
```

Comment:
What’s done it done, but in the future please add in alphabetical order (per comment).

It somewhat reduces merge conflicts if multiple people add or delete around the same time.

Reply:
Ah, that's embarrasing, here's the fix: https://github.com/stackrox/stackrox/pull/7321
And thanks for catching it!